lab_3_keywords_textrank/main.py

```diff
@@ -258,6 +258,23 @@ def __init__(self) -> None:
         self._position_weights = {}
 
     # Step 4.2
+    def add_edge_help(self, vertex: int, other_vertex: int) -> None:
```

Comment:
Why do you add the method to the class from LAB3?

Reply:
I'm not sure about adding a new method. I suppose, I added a space string to resolve style problem, which pycharm has told me about, and then I deleted that string. Then, probably, I commited a wrong file. However, it doesn't affect laboratory work #4 as I do not have to import class AdjacencyMatrixGraph, does it?

Reply:
remove